### PR TITLE
swiftdraw 0.12.0

### PR DESCRIPTION
**Pull request by swhitty**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/swiftdraw.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  end
+
+  test do
+    (testpath/"fish.svg").write <<~EOS
+      <?xml version="1.0" encoding="utf-8"?>
+      <svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="160" height="160">
+        <path d="m 80 20 a 50 50 0 1 0 50 50 h -50 z" fill="pink" stroke="black" stroke-width="2" transform="rotate(45, 80, 80)"/>
+      </svg>
+    EOS
+    system "#{bin}/swiftdraw", "#{testpath}/fish.svg", "--format", "sfsymbol"
```
Comment (carlocab) [inline Formula/swiftdraw.rb:24]: Does this produce a file? If so, can we check that it was created? You can do this with something like
```ruby
assert_path_exists testpath/"created-file-name"
```

Comment (swhitty) [reply Formula/swiftdraw.rb:24]: added

File: Formula/swiftdraw.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    assert_path_exists "#{testpath}/fish-symbol.svg"
```
Comment (carlocab) [inline Formula/swiftdraw.rb:25]:
```suggestion
    system bin/"swiftdraw", testpath/"fish.svg", "--format", "sfsymbol"
    assert_path_exists testpath/"fish-symbol.svg"
```
We don't need interpolation here.

**Review by carlocab — COMMENTED** [review]
Just one small nit, but otherwise looks good. Thanks!

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**